acceptance/setup/aio/pre-suite/010_Install.rb

```diff
@@ -44,3 +46,25 @@
     end
   end
 end
+
+step 'Install build dependencies on master' do
+
+ MASTER_PACKAGES = {
+    :redhat => [
+      'git',
+      'java-1.8.0-openjdk-devel',
+    ],
+  }
+  install_packages_on(master, MASTER_PACKAGES, :check_if_exists => true)
+end
+
+step 'Confirm SSH Forwarding Enabled on master' do
```

Comment:
Then you mean it's a clone via ssh, not via git.  git's own git:// wire protocol is distinct and doesn't need ssh setup.

I'm not convinced of the need for this extra ssh set-up.  Beaker's helper support https urls, https://github.com/puppetlabs/beaker/blob/master/lib/beaker/dsl/install_utils/foss_utils.rb#L42-L50 so we can continue to use that even with using the standard git helper.